elasticsearch/kibana/map.jinja

```diff
@@ -22,3 +15,31 @@
         'ssl': {}
     },
 }, grain='os_family', merge=salt['pillar.get']('kibana:lookup'), base='default') %}
+
+{% if elasticsearch.elastic_stack %}
+{% set kibana_conf = {
+    'config': {
+            'elasticsearch.url': 'http://localhost:9200',
+            'server.host': '127.0.0.1',
+            'server.port': 5601
+        },
+    'nginx_config': {
+            'server.ssl.certificate': '/etc/salt/ssl/certs/kibana.yourdomain.com.crt',
```

Comment:
Not sure I follow. My understanding is that this is adding the kibana cert and key to the nginx config, but the key names under nginx_config depend on the version of elasticsearch and that's why I have them there.